### PR TITLE
fix: python client examples

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

Fix python examples for client

## Links to any relevant issues

part of #388 

## Type of change

- Documentation Fix

## How the change has been tested

manually

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/examples/exchange/1_create_account.py [hunk 1]
```diff
@@ -19,8 +19,9 @@
         ".env STRONGHOLD_SNAPSHOT_PATH is undefined, see .env.example")
 if 'STRONGHOLD_PASSWORD' not in os.environ:
     raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
-if 'MNEMONIC' not in os.environ:
-    raise Exception(".env MNEMONIC is undefined, see .env.example")
+if 'NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1' not in os.environ:
+    raise Exception(
+        ".env NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1 is undefined, see .env.example")
```
Comment (thibault-martinez) [inline bindings/python/examples/exchange/1_create_account.py]: I think we have removed this from .env ?

Comment (Thoralf-M) [reply bindings/python/examples/exchange/1_create_account.py]: Yes only MNEMONIC and MNEMONIC_2 now

File: bindings/python/examples/exchange/1_create_account.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 # Store the mnemonic in the Stronghold snapshot, this only needs to be done once
 wallet.store_mnemonic(
-    os.environ['MNEMONIC'])
+    os.environ['NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1'])
```
Comment (thibault-martinez) [inline bindings/python/examples/exchange/1_create_account.py]: Same here

File: bindings/python/examples/exchange/1_create_account.py (hunk 1 above)
Comment (qrayven) [reply bindings/python/examples/exchange/1_create_account.py]: yeah, you're totally right, I  forgot to change my old env files.

File: bindings/python/examples/exchange/1_create_account.py (hunk 2 above)
Comment (qrayven) [reply bindings/python/examples/exchange/1_create_account.py]: already fixed

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**